e2e/wdio/core/tests/dynamic-page.e2e-spec.ts

```diff
@@ -0,0 +1,156 @@
+import {
+    click,
+    doesItExist,
+    getAttributeByName,
+    getElementArrayLength,
+    getElementClass,
+    isElementDisplayed,
+    pause,
+    refreshPage,
+    scrollIntoView,
+    waitForElDisplayed,
+    waitForPresent
+} from '../../driver/wdio';
+import { DynamicPagePo } from '../pages/dynamic-page.po';
+
+describe('dynamic side content test suite', function () {
+    const dynamicPagePage = new DynamicPagePo();
+    const {
+        button, pinButton, exitButton, saveButton, closeButton, cancelButton, rejectButton, acceptButton,
+        collapseButton, collapsibleHeader, columnLayoutExample, tabsExample, defaultExample,
+        responsiveExample, tab, dynamicPageContent, tabsContent, dynamicPage, flexileColumn
+    } = dynamicPagePage;
+
+    beforeAll(() => {
+        dynamicPagePage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForPresent(dynamicPagePage.root);
+        waitForElDisplayed(dynamicPagePage.title);
+    }, 1);
+
+    it('should check open-close', () => {
+        checkPageOpenClose(defaultExample);
```

Comment:
please take a look at comments from form-generator PR regarding structure and methods (read through all the comments there, a few are at the end)